Sources/SourceKitLSP/SourceKitServer.swift

```diff
@@ -2030,7 +2030,7 @@ extension SourceKitServer {
         fromRanges: [location.range]
       )
     }
-    return calls
```

Comment:
The call above is still picking a definition at random right? Given the sorting of SymbolOccurrence, should this also be by location and then name?

Reply:
Added a commit to the PR that gets rid of all the cases where we use `.first` on index results. Meant to do that anyway, so now is as good a time as any.